packages/contracts-bedrock/hardhat.config.ts

```diff
@@ -320,10 +320,6 @@ const config: HardhatUserConfig = {
       type: 'number',
       default: 6,
     },
-    fundDevAccounts: {
-      type: 'boolean',
-      default: false,
-    },
```

Comment:
Can you update the testnet setup tool to remove this flag too? `bedrock-regentool/regentool/config.py` uses this variable.

Reply:
Its not merged yet. https://github.com/ethereum-optimism/optimism/pull/3243

Reply:
I mean the script in the k8s repository